<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>compras</class>
 <widget class="QDialog" name="compras">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Compras</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../iconos/shopping_cart.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../iconos/shopping_cart.png</pixmap>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>551</width>
     <height>401</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <property name="elideMode">
    <enum>Qt::ElideNone</enum>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="icon">
     <iconset>
      <normaloff>../iconos/shopping_cart.png</normaloff>../iconos/shopping_cart.png</iconset>
    </attribute>
    <attribute name="title">
     <string>Carrito</string>
    </attribute>
    <widget class="QTableWidget" name="tabla_carrito">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>391</width>
       <height>131</height>
      </rect>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>130</number>
     </attribute>
     <column>
      <property name="text">
       <string>Nombre</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Marca</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Precio</string>
      </property>
     </column>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>71</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Productos</string>
     </property>
    </widget>
    <widget class="QPushButton" name="llevar_product_carrito">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Poner en el carrito</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../iconos/add_32.png</normaloff>../iconos/add_32.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="icon">
     <iconset>
      <normaloff>../iconos/calculator.png</normaloff>../iconos/calculator.png</iconset>
    </attribute>
    <attribute name="title">
     <string>Caja</string>
    </attribute>
    <widget class="QLabel" name="lb_subtotal">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>230</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lb_impuesto">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>270</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Subtotal</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>280</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Impuesto</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_total_factura">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>310</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>320</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Total en factura</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pagar_factura">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>310</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Pagar</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../iconos/credit-card.png</normaloff>../iconos/credit-card.png</iconset>
     </property>
    </widget>
    <widget class="QTableWidget" name="tabla_caja">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>341</width>
       <height>131</height>
      </rect>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>114</number>
     </attribute>
     <column>
      <property name="text">
       <string>Nombre</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Marca</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Precio</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>20</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Remover del carrito</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../iconos/close_delete.png</normaloff>../iconos/close_delete.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="calcular_factura">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>90</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Calcular factura</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../iconos/calculator.png</normaloff>../iconos/calculator.png</iconset>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
